### ISSUE TITLE
[CSBS] missing support for api parameters

### ISSUE THREAD

**Issue by sfhl** [opentelekomcloud/terraform-provider-opentelekomcloud#578]
Hi there,
for csbc policies, there is an additonal option to configure the retention of n numbers of daily/weekly/monthly/yearly backups:
  - day_backups
  - week_backups
  - month_backups
  - year_backups

It would be great if this would be supported by the provider.
  See also: https://docs.otc.t-systems.com/en-us/api/csbs/en-us_topic_0059304223.html

### Terraform Version
```
terraform version
Terraform v0.12.12
+ provider.opentelekomcloud v1.19.0
```

### Affected Resource(s)
Please list the resources as a list, for example:
- opentelekomcloud_csbs_backup_policy_v1

### Expected Behavior
Provider should support corresponding attributes.

### Actual Behavior
Provider does not support corresponding attributes.

### Steps to Reproduce
1. `terraform apply`

**Comment by outcatcher** [comment]
Will be fixed after SDK migration

**Comment by outcatcher** [comment]
Implemented in release v1.21.5
